sourcegraph/sourcegraph
# Code Insights: Add compute insights to the standalone and the dashboard pages

Closes https://github.com/sourcegraph/sourcegraph/issues/38344
Closes https://github.com/sourcegraph/sourcegraph/issues/37969


## Test plan
- Make sure that you can see compute insights on the dashboard page
- All filters should work properly (search context and repositories filters)
- Click on the card title you should see the standalone insight page with compute powered insight chart
